tests/test_core.py

```diff
@@ -896,6 +896,18 @@ def test_emmalloc_memalign_corruption(self, *args):
     self.set_setting('MALLOC', 'emmalloc')
     self.do_run_in_out_file_test('tests', 'core', 'emmalloc_memalign_corruption.cpp')
 
+  @no_lsan('This test does not free allocated memory by design')
+  @no_asan('ASan does not support custom memory allocators')
+  @parameterized({
+    '2gb': [],
+    # TODO when Node.js supports 4GB Wasm memories:
+    #    '4gb': ['-s', 'MAXIMUM_MEMORY=4GB']
+  })
+  def test_emmalloc_memgrowth(self, *args):
+    self.emcc_args += ['-s', 'MALLOC=emmalloc', '-s', 'ALLOW_MEMORY_GROWTH=1', '-s', 'ABORTING_MALLOC=0', '-s', 'ASSERTIONS=2', '-s', 'MINIMAL_RUNTIME=1'] + list(args)
+    self.do_run(open(path_from_root('tests', 'core', 'mem_growth.cpp')).read(),
+                open(path_from_root('tests', 'core', 'mem_growth_4gb.out' if 'MAXIMUM_MEMORY=4GB' in args else 'mem_growth_2gb.out')).read())
```

Comment:
Running these 2GB-allocating tests in core is risky - we put them in `browser`, which runs serially, and with prefixes so they run last. See `test_zzz_zzz_2gb_fail` there.

Reply:
Check.